spec/spec_helper.rb

```diff
@@ -3,6 +3,13 @@
 require 'timecop'
 require 'webmock'
 require 'vcr'
+require 'dotenv/load'
+
+# ENV['LEANPLUM_PRODUCTION_KEY']='prod_I02AwsAsRGi7g0NNc2YA2lyUceOROrkDLDHcheWBNk4'
+# ENV['LEANPLUM_APP_ID']='app_bQZEF70IGUuPPYSLcZEON5nlihNa9N9GukO93mSpT48'
```

Comment:
you probably shouldn't commit valid credentials to a public repo....

Reply:
oops. I hate it when that happens. I added dotenv gem to repo to handle this.

Reply:
feel free to add `dotenv` and tell people how to run the specs with an env file - that's basically what i have done while testing as well.  but you should at this point delete this app from leanplum, because this will never come out of the commit history.